src/test/scala/is/hail/expr/ir/CompileSuite.scala

```diff
@@ -674,4 +674,32 @@ class CompileSuite {
       assert(SafeRow(TStruct("foo" -> TInt32()), region, off).get(0) == 6)
     }
   }
+
+  @Test def testUniroot() {
+    val ir = Uniroot("x",
+      ApplyBinaryPrimOp(Add(), Ref("x", TFloat64()), F64(3)),
+      F64(-6), F64(0))
+    val fb = EmitFunctionBuilder[Region, Double]
+    doit(ir, fb)
+    val f = fb.result()()
+    Region.scoped { region =>
+      assert(f(region) == -3)
```

Comment:
I think it makes sense for all new IR tests to use `TestUtils.testEvalsTo`, which checks that compiler, interpreter, and expected all agree, and keeps the tests readable.